cpp/include/cudf/functions.h

```diff
@@ -894,6 +894,60 @@ gdf_error gdf_group_by_count(int ncols,                    // # columns
                              gdf_column* out_col_agg,      //aggregation result
                              gdf_context* ctxt);            //struct with additional info: bool is_sorted, flag_sort_or_hash, bool flag_count_distinct
 
+/* --------------------------------------------------------------------------*/
+  /**
+   * @brief Given a set of columns, of which a subset of these are defined to be the group by columns,
```

Comment:
I find it grammatically easier to refer to the columns you're "grouping by" as the "key" columns rather than "group by" columns.

Reply:
@jrhemstad : Among database people, "key" is commonly used w.r.t. tables or joins, not grouping.

Reply:
But they are "key"s. Groupby is just a "reduce-by-key", where the "groupby" columns are the "key" columns. Pandas users I've talked to also refer to these as the key columns.

Reply:
And we use this terminology ("keys") elsewhere in cuDF.